sqlglot/dialects/bigquery.py

```diff
@@ -293,6 +293,11 @@ class BigQuery(Dialect):
     # All set operations require either a DISTINCT or ALL specifier
     SET_OP_DISTINCT_BY_DEFAULT = dict.fromkeys((exp.Except, exp.Intersect, exp.Union), None)
 
+    ANNOTATORS = {
+        **Dialect.ANNOTATORS,
+        exp.Sign: lambda self, e: self._annotate_by_args(e, "this"),
```

Comment:
What do other dialects that support `SIGN` do? Should we annotate this as `INT` or something by default?

Reply:
It's already annotated as such in the base annotators, under `Type.INT`

Reply:
Regarding the other dialects it's a mix of INT/DOUBLE/input, so I left it as is for now.

Reply:
Sounds good, thanks for double checking this 👍